src/parquet/column/test-util.h

```diff
@@ -174,9 +174,34 @@ class DataPageBuilder {
     buffer_size_ += levels_footprint;
   }
 };
-
 } // namespace test
 
+static inline void InitDataPage(const parquet::Statistics& stat,
+    parquet::DataPageHeader& data_page, int nvalues) {
+  data_page.encoding = parquet::Encoding::PLAIN;
+  data_page.definition_level_encoding = parquet::Encoding::RLE;
+  data_page.repetition_level_encoding = parquet::Encoding::RLE;
+  data_page.num_values = nvalues;
+  data_page.__set_statistics(stat);
+}
+
+static inline void InitStats(size_t stat_size, parquet::Statistics& stat) {
+  std::vector<char> stat_buffer;
+  stat_buffer.resize(stat_size);
+  for (int i = 0; i < stat_size; i++) {
+    (reinterpret_cast<uint8_t*>(stat_buffer.data()))[i] = i % 255;
+  }
+  stat.__set_max(std::string(stat_buffer.data(), stat_size));
```

Comment:
This function is doing two things -- generating random string data and populating the `parquet::Statistics` object -- it would better to have two functions that do each of these tasks, but it's not a big deal right now